code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -66,6 +68,22 @@
 	internal_organs += new /obj/item/organ/alien/eggsac
 	..()
 
+/mob/living/carbon/alien/humanoid/royal/queen/proc/game_end()
+	if(stat != DEAD)
+		SSshuttle.clearHostileEnvironment(src)
+		if(EMERGENCY_IDLE_OR_RECALLED)
+			priority_announce("Xenomorph infestation detected: crisis shuttle protocols activated - jamming recall signals across all frequencies. P.S.: Bring back a live one please.")
```

Comment:
Message seems misleading since there's no requirement or bonus for bringing one back alive.

Reply:
It's just a reference to Aliens, specifically the megacorporaiton Weyland-Yutano.

Reply:
I'm aware.

Reply:
Alright, i removed it.